feat: release 0.11.0 (#109)
ConduitIO/conduit-site: docs/features/opencdc-record.mdx

to indicate what they refer to, preventing accidental clashes. Here are the ones
you can find:

### `opencdc.version`

Contains the version of the OpenCDC format (e.g., "v1"). This field exists to
ensure the OpenCDC format version can be easily identified in case the record
gets marshaled into a different untyped format (e.g. JSON).

> ```json5

Reviewer: This `json5` thingy doesn't seem to work for me, while the `json` directive used above works. Maybe rename `json5` to `json` for nicer code highlighting?

}
```

### `opencdc.collection`

Contains the name of the collection from which the record originated and/or
where it should be written to.

:::note
It's up to the connector to populate this field. In other words, not all records
may have this field.
:::

```json5
{
    // other record fields
    "metadata": {
        "opencdc.collection": "employees",
        // rest of metadata
    },
    // other record fields
}
```

### `opencdc.key.schema.*`

#### `opencdc.key.schema.subject`, `opencdc.key.schema.version`

> Contains the subject and version of the schema for the records's `.Key` field.

Reviewer 2: ```suggestion
Contains the subject and version of the schema for the records' `.Key` field.
```

:::note
This field will only be populated when using structured data.
:::

```json5
{
    // other record fields
    "metadata": {
        "opencdc.key.schema.subject": "employees.key.v1",

>         "opencdc.key.schema.version": "1.0.0",

Reviewer 2: schema version is an integer, so examples should be `1`, `2`, ...
```suggestion
        "opencdc.key.schema.version": "1",
```

        // rest of metadata
    },
    // other record fields
}
```

### `opencdc.payload.schema.*`

#### `opencdc.payload.schema.subject`, `opencdc.payload.schema.version`

> Contains the subject and version of the schema for the records' `.Payload` field.

Reviewer: ```suggestion
Contains the subject and version of the schema for the records' `.Payload.After` and `.Paylod.Before` fields.
```

> Contains the subject and version of the schema for the records's `.Payload` field.

Reviewer 2: ```suggestion
Contains the subject and version of the schema for the records' `.Payload` field.
```

:::note
This field will only be populated when using structured data.
:::

```json5
{
    // other record fields
    "metadata": {
        "opencdc.payload.schema.subject": "employees.value.v2",

>         "opencdc.payload.schema.version": "2.1.2",

Reviewer 2: same here
```suggestion
        "opencdc.payload.schema.version": "2",
```
